app/controllers/users_controller.rb

```diff
@@ -113,6 +113,18 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    user = find_resource(:edit_users)
```

Comment:
Move this line into the rescue block, this is ajax and ordinary exceptions are not what we want.